docs/querying/basics.md

```diff
@@ -87,16 +88,53 @@ Examples:
     0x8f
     -Inf
     NaN
-	
 
-As of version 2.54, float literals can also be represented using the syntax of time durations, where the time duration is converted into a float value corresponding to the number of seconds the time duration represents. This is an experimental feature and might still change.
+Additionally, underscores (`_`) can be used in between decimal or hexadecimal
+digits to improve readability.
 
 Examples:
 
-    1s # Equivalent to 1.0
-    2m # Equivalent to 120.0
-    1ms # Equivalent to 0.001
- 
+    1_000_000
+	.123_456_789
```

Comment:
There's an indentation inconsistency regarding tabs vs. spaces here. Same in the sections below.

Reply:
I have untabified the whole file. I hope it's fine now.